File: src/test/java/com/codingbat/warmup1/MonkeyTroubleTest.java
package com.codingbat.warmup1;

import static org.junit.Assert.assertFalse;
import static org.junit.Assert.assertTrue;

import org.junit.Test;

public class MonkeyTroubleTest {

  @Test
  public void willBeInTroubleIfBothMonkeysAreSmiling() {
    assertTrue(MonkeyTrouble.monkeyTrouble(true, true));
  }

  @Test
  public void willBeInTroubleIfNeitherMonkeyIsSmiling() {
    assertTrue(MonkeyTrouble.monkeyTrouble(false, false));
  }

  @Test
  public void shouldNotBeInTroubleIfFirstMonkeyIsSmiling() {
    assertFalse(MonkeyTrouble.monkeyTrouble(true, false));
  }

  @Test
  public void shouldNotBeInTroubleIfSecondMonkeyIsSmiling() {
    assertFalse(MonkeyTrouble.monkeyTrouble(false, true));
  }
}

File: src/test/kotlin/com/codingbat/warmup1/MonkeyTroubleTest.kt
package com.codingbat.warmup1

import org.junit.Test
import kotlin.test.assertFalse
import kotlin.test.assertTrue

class MonkeyTroubleTestKt {
  @Test
  fun willBeInTroubleIfBothMonkeysAreSmiling() {
    assertTrue(monkeyTrouble(true, true))
  }

  @Test
  fun willBeInTroubleIfNeitherMonkeyIsSmiling() {
    assertTrue(monkeyTrouble(false, false))
  }

  @Test
  fun shouldNotBeInTroubleIfFirstMonkeyIsSmiling() {
    assertFalse(monkeyTrouble(true, false))
  }

  @Test
  fun shouldNotBeInTroubleIfSecondMonkeyIsSmiling() {
    assertFalse(monkeyTrouble(false, true))
  }
}
